packages/api-explorer/src/PathUrl.jsx

```diff
@@ -9,11 +9,20 @@ const { Operation } = Oas;
 const extensions = require('@readme/oas-extensions');
 
 function splitPath(path) {
+  const keys = [];
   return path
     .split(/({.+?})/)
     .filter(Boolean)
     .map(part => {
       return { type: part.match(/[{}]/) ? 'variable' : 'text', value: part.replace(/[{}]/g, '') };
+    })
+    .map(part => {
+      // To ensure unique keys, we're going to create a key with
+      // the value concatenated to the count of the value.
+      const keyCount = keys.filter(key => key === part.value).length;
+      keys.push(part.value);
+      part.key = `${part.value}-${keyCount}`;
```

Comment:
You could also, instead of finding a count, use the `i` value of the current iteration and add that.

```
.map((part, i) => {
```

That may negate the need for the array/Set altogether?

Reply:
Huh hadn't thought of that! I guess there's no reason to use a unique index specific to the value itself. I'll simplify that!